fl4health/parameter_exchange/layer_exchanger.py

```diff
@@ -9,7 +9,7 @@
 from fl4health.parameter_exchange.partial_parameter_exchanger import PartialParameterExchanger
 
 TorchModule = TypeVar("TorchModule", bound=nn.Module)
-LayerSelectionFunction = Callable[[nn.Module, nn.Module], Tuple[NDArrays, List[str]]]
+LayerSelectionFunction = Callable[[nn.Module, Optional[nn.Module]], Tuple[NDArrays, List[str]]]
```

Comment:
This change here is simply because I realized the old type annotation was not quite accurate.